src/cordova/build.action.mjs

```diff
@@ -70,7 +70,11 @@ export async function main(...parameters) {
         '--gradleArg=-PcdvBuildMultipleApks=true',
       ];
     }
+
+    cordova.on('verbose', console.debug);
+
     await cordova.compile({
+      verbose: buildMode === 'debug',
```

Comment:
I'm not sure this makes sense.  Why would we only want detailed build logs when building the debug version?  I think they're probably independent questions.

Reply:
Maybe add an additional argument to action? `npm run action cordova/build android -- -verbose`

Reply:
I agree with Ben. Verbose and debug are separate things.

Perhaps we should just forward the extra params from the `npm run action` to the cordova call